src/hotspot/share/memory/metaspace/metaspaceReporter.cpp

```diff
@@ -1,6 +1,6 @@
 /*
- * Copyright (c) 2018, 2020, Oracle and/or its affiliates. All rights reserved.
- * Copyright (c) 2018, 2020 SAP SE. All rights reserved.
+ * Copyright (c) 2018, 2021, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2018, 2021 SAP SE. All rights reserved.
```

Comment:
Please note that while Oracle requests/requires that it's copyright line gets updated each year a file is modified, it is not clear whether that extends to any other copyright lines that might appear in the file. In general for companies other than Oracle, only a committer from that company should update a copyright line for that company.

Thanks,
David

Reply:
Changed. Thanks for pointing out this.

Reply:
I would have been fine with the copyright change :)